build-support/docker/travis_ci_py27_ucs2/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I'm not certain this was the best approach to create a new Dockerfile.

The goal is to install Python 2.7 via Pyenv using the env var `PYTHON_CONFIGURE_OPTS --enable-unicode=ucs2` (see lines 7 - 16). I believe this can all be achieved via the Docker CLI within `.travis.yml`, and I started with this approach, but then broke it out into this Dockerfile to keep `.travis.yml` cleaner.

However, I'm not convinced this is the better solution. It duplicates lines 5 and 17-35 from `travis_ci/Dockerfile`, and violates the principle of locality as this file is only useful for that one specific CI shard `py27_linux_build_wheels_ucs2` (which we're going to remove within 2 major releases once we drop Py2).

--

If we stick with Dockerfile, I'll fix the bad copyright year!